package.json

```diff
@@ -29,6 +29,9 @@
     "test:scss": "stylelint \"source/sass/**/*.scss\" \"source/js/**/*.scss\" --syntax scss",
     "test:scss:styleguide:color": "stylelint \"source/sass/**/*.scss\" \"source/js/**/*.scss\" \"!source/sass/**/_colors.scss\" --syntax scss --config .stylelintrc-colors.js",
     "test:css": "stylelint \"network-api/networkapi/{,!(frontend)/**/}*.css\" --syntax scss",
+    "selenium": "run-p --race server test:site:selenium",
+    "selenium:run": "mocha ./test/selenium --timeout 60000",
+    "selenium:site:run": "wait-on tcp:8000 && npm run test:selenium",
```

Comment:
Is this task `test:selenium` missing too?

Reply:
no, because there's a `test:*` as part of the main test running, and selenium should _not_ run on every test run.